pandapipes/test/pipeflow_internals/test_inservice.py

```diff
@@ -210,7 +210,7 @@ def test_connectivity_hydraulic(create_test_net, use_numba):
     pandapipes.pipeflow(net, iter=100, tol_p=1e-7, tol_v=1e-7, friction_model="nikuradse",
                         use_numba=use_numba)
 
-    assert np.all(np.isnan(net.res_junction.loc[[2, 5, 6], :].values))
+    assert np.all(np.isnan(net.res_junction.p_bar.loc[[2, 5, 6]].values))
```

Comment:
Good point, as far as I can recall, the former is more performant which is why I usually implement it... but it might depend on the circumstances (size of DF, different dtypes etc.)